include/gatekeeper_ipip.h

```diff
@@ -33,4 +33,7 @@ struct ipip_tunnel_info {
 int encapsulate(struct rte_mbuf *pkt, uint8_t priority,
 	struct ipip_tunnel_info *info);
 
+int decapsulate(struct rte_mbuf *pkt, uint8_t *priority,
+	struct ipip_tunnel_info *info);
```

Comment:
`decapsulate()` does more than just remove headers, and a comment explaining its details is warranted. Suggestion:

```
/*
 * Remove the Ethernet header and the outer IP header of an IP in IP tunnel, and
 *	copy some fields into @priority and @info.
 * @priority receives the priority assigned by Gatekeeper servers.
 * @info is ONLY updated if @priority is greater or equal to 2.
 *
 * RETURN
 *	If the packet is not a well-formed IP in IP packet, it returns -1 and
 *		doesn't update @priority and @info.
 *	Otherwise it returns zero.
 */
```